src/overlays/actors/ovl_En_Az/z_en_az.h

```diff
@@ -8,9 +8,44 @@ struct EnAz;
 typedef void (*EnAzActionFunc)(struct EnAz*, PlayState*);
 
 typedef struct EnAz {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ EnAzActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x290];
+    /* 0x0000 */ Actor actor;
+    /* 0x0144 */ EnAzActionFunc actionFunc;
+    /* 0x0148 */ SkelAnime skelAnime;
+    /* 0x018C */ ColliderCylinder collider;
+    /* 0x01D8 */ Vec3s jointTable[24];
+    /* 0x0268 */ Vec3s morphTable[24];
+    /* 0x02F8 */ s16 unk_2F8;
+    /* 0x02FA */ s16 unk_2FA; // cutscene state?
+    /* 0x02FC */ s32 animIndex;
+    /* 0x0300 */ ActorPathing unk_300;
+    /* 0x036C */ f32 unk_36C;
+    /* 0x0370 */ char unk370[4];
+    /* 0x0374 */ u16 unk_374; // flags of some sort
+    /* 0x0376 */ u16 unk_376; // flags of some sort
+    /* 0x0378 */ u8 unk_378; // cutscene state?
+    /* 0x037A */ s16 unk_37A;
+    /* 0x037C */ s16 unk_37C;
+    /* 0x037E */ s16 unk_37E;
+    /* 0x0380 */ s16 unk_380;
+    /* 0x0382 */ s16 unk_382;
+    /* 0x0384 */ s16 unk_384;
+    /* 0x0388 */ struct EnAz* brother;
+    /* 0x038C */ char unk38C[0x10];
```

Comment:
```suggestion
    /* 0x038C */ UNK_TYPE1 unk38C[0x10];
```
We use this for unused pads, could you fill these out for the struct?